plansys2_bt_actions/include/plansys2_bt_actions/BTActionNode.hpp

```diff
@@ -36,14 +36,11 @@ class BtActionNode : public BT::ActionNodeBase
     const std::string & xml_tag_name,
     const std::string & action_name,
     const BT::NodeConfiguration & conf)
-  : BT::ActionNodeBase(xml_tag_name, conf), action_name_(action_name)
+  : BT::ActionNodeBase(xml_tag_name, conf), action_name_(action_name),
+    return_failure_(false)
```

Comment:
Is it necessary to be a class member? `on_new_goal_received` could better return bool.